### PR TITLE
Ongoing testing

### PR DESCRIPTION
**Pull request by misterbisson**
...continued work from https://github.com/heyawhite/happy-randomizer-terraform/pull/1

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -5,13 +5,6 @@ provider "triton" {
     # key_id = "${var.triton_key_id}"
 }
 
-#
-# Details about all deployments of this application
-#
-data "triton_network" "public" {
```
Comment (misterbisson) [inline main.tf]: I'd like to consolidate this with the other vars, but there's clearly something about TF I don't understand yet.

File: main.tf [hunk 2]
```diff
@@ -48,9 +41,9 @@ resource "triton_machine" "green_machine" {
     name = "green_happy_${count.index + 1}"
     package = "g4-highcpu-2G"
     image = "${data.triton_image.green_image.id}"
-    networks = ["${data.triton_network.public.id}"]
+    networks = "${ var.service_networks }"
     cns {
-        services = ["${var.production == "green" ? var.service_name : "staging-${var.service_name}" }", "blue-${var.service_name}"]
+        services = ["${var.service_production == "green" ? var.service_name : "staging-${var.service_name}" }", "blue-${var.service_name}"]
```
Comment (misterbisson) [inline main.tf]: `"blue-${var.service_name}"]`

That was some copypasta

**Comment by alexandrascript** [comment]
Is this still on-going? Or available for review/merging?

**Comment by misterbisson** [comment]
Merge away!

**Pull request merged**